src/chat-bubble/internal.tsx

```diff
@@ -54,12 +57,15 @@ export default function InternalChatBubble({
         className={clsx(styles["message-area"], styles[`chat-bubble-type-${type}`], {
           [styles["with-loading-bar"]]: showLoadingBar,
```

Comment:
I think you removed the `with-loading-bar` style in the scss. Is it still relevant here?